deploy-manage/remote-clusters/ec-enable-ccs-for-eck.md

```diff
@@ -12,84 +12,137 @@ products:
 
-### Establish trust in the {{ech}} cluster [ec_establish_trust_in_the_elasticsearch_service_cluster]
+### Establish trust in the ECH cluster [ec_establish_trust_in_the_elasticsearch_service_cluster]
 
-1. Save the ECK CA certificate to a file. For a cluster named `quickstart`, run:
+1. Save the {{es}} transport CA certificate of your ECK deployment. For an {{es}} cluster named `quickstart`, run:
 
     ```sh
-    kubectl get secret quickstart-es-transport-certs-public -o go-template='{{index .data "ca.crt" | base64decode}}' > eck.ca.crt
+    kubectl get secret quickstart-es-transport-certs-public -o go-template='{{index .data "ca.crt" | base64decode}}' > eck-ca.crt
     ```
 
+    This command saves the certificate to `eck-ca.crt`.
 
-1. Update the trust settings for the {{ech}} deployment. Follow the steps provided in [Access clusters of a self-managed environment](ec-remote-cluster-self-managed.md), and specifically the first three steps in **Specify the deployments trusted to be used as remote clusters** using TLS certificate as security model.
+2. Update the trust settings for the {{ech}} deployment:
 
-    * Use the certificate file saved in the first step.
-    * Select the {{ecloud}} pattern and enter `default.es.local` for the `Scope ID`.
+    1. From the **Security** menu, select **Remote Connections > Add trusted environment**, choose **Self-managed**, and click **Next**.
 
-2. Select `Save` and then download the CA Certificate and `trust.yml` file. These files can also be retrieved in the `Security` page of the deployment. You will use these files in the next set of steps.
+    2. Select **Certificates** as authentication mechanism and click **Next**.
 
+    3. In **Add trusted CA certificate**, upload the `eck-ca.crt` file retrieved in the previous step.
+
+    4. In **Select trusted clusters**, configure the following:
+        * Select **Trust clusters whose Common Name follows the Elastic pattern**.
+        * For **Scope ID**, enter `<kubernetes-namespace>.es.local`, replacing `<kubernetes-namespace>` with the namespace of your ECK cluster.
+        * In **Trust**, select **All deployments**.
+
+    5. In **Name the environment**, enter a name for the trusted environment. That name will appear in the trust summary of your deployment’s **Security** page.
+
+    6. Select **Create trust** to complete the configuration.
+
+    7. On the confirmation screen, when prompted **Have you already set up trust from the other environment?**, select **No, I have NOT set up trust from the other environment yet**. Download both the ECH deployment CA certificate and the `trust.yml` file. These files can also be retrieved from the **Security** page of the deployment. You’ll use these files to configure trust in the ECK deployment.
 
 ### Establish trust in the ECK cluster [ec_establish_trust_in_the_eck_cluster]
```

Comment:
:2c: I'd split out editing trust.yml into its own procedure because it's too long for a single step